### DISCUSSION TITLE
Custom SegmentedControl

### DISCUSSION THREAD

**Discussion by thekra** [xmartlabs/Eureka#2140] [Q&A]
I was wondering if I could use my own custom segmented control, and if it is possible how do I implement it ?

**Comment by mats-claassen** [comment]
You can create a nib file with your custom design and custom segmented control. You can then either set the class of the cell to `SegmentedCell` or a concrete (non-generic) subclass and connect the outlets to it. 

Or you can just create a custom cell and row. Have a look at the Readme for instructions.

**Reply by thekra** [reply]
Thank you for your reply, I eventually created a nib file and it worked just fine
